service/src/test/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneServiceTest.java

```diff
@@ -196,7 +192,51 @@ void startLandingZoneCreationJob_JobIsSubmitted() {
       landingZoneService.startLandingZoneCreationJob(
           bearerToken, "newJobId", landingZoneRequest, "create-result");
     }
+    verify(mockJobBuilder, times(1))
+        .addParameter(eq(LandingZoneFlightMapKeys.LANDING_ZONE_ID), captorLandingZoneId.capture());
+    assertNotEquals(landingZoneId, captorLandingZoneId.getValue());
+    verify(landingZoneJobService, times(1)).newJob();
+    verify(mockJobBuilder, times(1)).submit();
+  }
+
+  @Test
+  void startLandingZoneCreationJob_WithLandingZoneId_JobIsSubmitted() {
```

Comment:
I think we are missing a test to validate that when the landing zone id is not provided in the request, a new UUID is generated. I realized that this test in theory should've been there already, but now that we made the lz id optional is important to have a test that validates the default behavior.

Reply:
I think the original test point `startLandingZoneCreationJob_JobIsSubmitted` covers that, but perhaps it could be renamed for clarity.